pkg/k8s/watchers/endpoint.go

```diff
@@ -156,5 +155,6 @@ func (k *K8sWatcher) addKubeAPIServerServiceEPs(ep *slim_corev1.Endpoints) {
 		}
 	}
 
-	k.handleKubeAPIServerServiceEPChanges(desiredIPs)
+	rid := ipcacheTypes.NewResourceID("ep", ep.ObjectMeta.GetNamespace(), ep.ObjectMeta.GetName())
```

Comment:
I guess we'll somehow need to keep these resource types constant ("ep") so that in the future, users don't mistakenly use a different string to represent the same resource type. Defining them in the ipcache types pkg seems like the most obvious place.

Reply:
The intent here is to use the canonical short name for the resource, I wonder if we already have constants for those.. Will look into it.

Reply:
Ah but I take your point, there are cases that don't come from k8s. So defining these constants in ipcache package will ensure there is one canonical set of non-conflicting names. I'll put it there.